engine/apps/alerts/models/alert_receive_channel.py

```diff
@@ -422,8 +422,9 @@ def emojized_verbal_name(self):
 
     @property
     def new_incidents_web_link(self):
-        return urljoin(
-            self.organization.web_link, f"?page=incidents&integration={self.public_primary_key}&status=0&p=1"
```

Comment:
this previously had a bug in it, the `?page=<page>` URL format is no longer supported, this will now point to the `alert-groups` page w/ the necessary filtering query params